src/vs/workbench/contrib/debug/browser/debugEditorContribution.ts

```diff
@@ -323,7 +323,6 @@ export class DebugEditorContribution implements IDebugEditorContribution {
 					this.altPressed = true;
 					const debugHoverWasVisible = this.hoverWidget.isVisible();
 					this.hoverWidget.hide();
-					this.enableEditorHover();
```

Comment:
Would we still need this so that hover works automatically when holding alt then moving the mouse somewhere else?

Reply:
Hmm 'showEditorHover' invokes it since I added it there for fallback. So this will result in upadating the editor twice . Would that be necessary?

Reply:
showEditorHover isn't called in the scenario I'm thinking of (I think), because the editor normally manages its own hover

Reply:
Oh I see why it still works. But it's a little confusing to not update this when the alt key is pressed. For example then it depends on the timeout for the debug hover, not the timeout for the editor hover, and it feels like the code could be fragile. I think it's fine if the enable happens twice.